samples/visual-indicator/nodejs/appPackage/manifest.json

```diff
@@ -0,0 +1,88 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/teams/v1.15/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.15",
+  "version": "1.0.0",
+  "id": "<<YOUR-MICROSOFT-APP-ID>>",
+  "packageName": "com.microsoft.teams.visualindicator",
+  "developer": {
+    "name": "Microsoft",
+    "websiteUrl": "https://microsoft.com",
+    "privacyUrl": "https://microsoft.com/privacy",
+    "termsOfUseUrl": "https://microsoft.com/termsofuse"
+  },
+  "icons": {
+    "color": "color.png",
+    "outline": "outline.png"
+  },
+  "name": {
+    "short": "Visual Indicator",
+    "full": "Visual Indicator In Meetings"
+  },
+  "description": {
+    "short": "Visual Indicator In-Meeting Notification demo",
+    "full": "This sample app shows how to send visual indicator in meetings using targeted notifications API."
```

Comment:
This sample app shows how to send visual indicator in Teams meetings using targeted notifications API.

Reply:
fixed as per comment